plugins/catalog-customized/src/plugin.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { catalogPlugin } from '@backstage/plugin-catalog';
+
+import {
+  EntityOwnerPicker,
+  EntityTypePicker,
+  UserListPicker,
+} from './components';
+
+catalogPlugin.reconfigure({
+  EntityOwnerPicker,
+  EntityTypePicker,
+  UserListPicker,
+  createButtonTitle: 'Maybe Create',
+  showButtonText: 'Mine catalog entities',
+});
```

Comment:
Sidenote that we don't need to deal with just yet: I'd like to figure out if and how we can make this async or otherwise have it not blow up the main bundle size if a lot of customizations are used.

Reply:
I can remove it back, I added it, as you mentioned in the previous feedback to add more customization with components as well.